Documentation/coding-guidelines/adding-api-guidelines.md

```diff
@@ -0,0 +1,124 @@
+    - Update framework section in `<Library>\src\project.json` (ex: [System.Runtime\src\project.json](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/src/project.json#L3))
+    - Update build configurations in `<Library>\src\<Library>.builds` if necessary (see [Project configuration conventions](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#project-configuration-conventions))
+      - The default build configuration(i.e. TargetGroup excluded) should build the latest version
+      - If you need a live build of the older versions they need to be added as a new configuration
+        - Typically only done for libraries not in netstandard that have larger amounts of code
+          that we want to update outside of servicing events.
+  - Make your code changes.
+
+**Update pkg**
+ - If changing the target framework
+    - Update SupportedFrameworks metadata on the ref ProjectReference to declare the set of
+      concrete of platforms you expect your library to support. (see [Specific platform mappings](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#nuget)). Generally
+      will be a combination of netcoreapp1.x, netfx46x and/or uap10.x.
+    - If package is not split (i.e. only one pkgproj, ex [pkg\System.Diagnostics.Process.pkgproj](https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.Process/pkg/System.Diagnostics.Process.pkgproj)) then it should already have a `ProjectReference` the `<Library>\src\<Library>.builds` file and so there is no additional changes needed in the pkgproj.
+    - If package is RID split (i.e. has multple pkgprojs, ex: [pkg\any\System.Runtime.pkgproj](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/pkg/any/System.Runtime.pkgproj) and [pkg\aot\System.Runtime](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/pkg/aot/System.Runtime.pkgproj))
+      - Update the ProjectReferences in the respective pkgproj's to align with the build configurations in the
+      `<Library>\src\<Library>.builds` file. This may entail adding new references or removing references according to whatever changes were made to the .builds file.
+  - If assembly or package version is updated the package index needs to be updated by running
+    msbuild `<Library>/pkg/<Library>.pkgproj /t:UpdatePackageIndex`
+
+**Update tests**
+  - If changing target framework
+    - Update `NugetTargetMoniker` in `<Library>\tests\<Library>.csproj` (ex: [tests\System.Runtime.csproj](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/System.Runtime.Tests.csproj#L13))
+    - Update framework section in `<Library>\tests\project.json` (ex: [System.Runtime\tests\project.json](https://github.com/dotnet/corefx/blob/master/src/System.Runtime/tests/project.json#L30))
```

Comment:
(General) whereever you use the word _Update_ can you please replace with how it's being updated.

Example: instead of "Update framework section" use "Append the new TFM to the frameworks  section"